src/SFA.DAS.EmployerAccounts.Web/Views/Home/Index.cshtml

```diff
@@ -8,72 +8,76 @@
 @{ ViewBag.HideNav = true; }
 @{ ViewBag.HideAccounts = "true"; }
 @{ ViewBag.GaData.Vpv = "/page-auth-homepage"; }
+@{Layout = "~/Views/Shared/_Layout_CDN.cshtml"; }
 
 @if (!string.IsNullOrWhiteSpace(Model.Data.ErrorMessage))
 {
-    <div class="grid-row">
-        <div class="column-full">
-            <div class="panel panel-error">
-                @Model.Data.ErrorMessage
-            </div>
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+        <div class="panel panel-error">
+            @Model.Data.ErrorMessage
         </div>
     </div>
+</div>
 }
-<div class="grid-row">
-    <div class="column-full">
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
         @if (Model.Data.Accounts.AccountList.Count != 0)
         {
-            <h1 class="heading-xlarge" id="Levy-accounts">Your accounts</h1>
-            <p>Select an account or add a new one.</p>
-            <table class="responsive">
-                <thead>
-                    <tr>
-                        <th scope="col" class="tw-30">Account name</th>
-                        <th scope="col">What you can do</th>
-                        <th scope="col">
-                            <span class="vh">Select account</span>
-                        </th>
-                    </tr>
-                </thead>
-                <tbody>
-                    @foreach (var account in Model.Data.Accounts.AccountList)
+        <h1 class="govuk-heading-xl" id="Levy-accounts">Your accounts</h1>
+
+        <p class="govuk-body govuk-!-margin-bottom-6">Select an account or add a new one.</p>
+
+        <table class="govuk-table">

```

Comment:
This table needs an extra class of 'das-table--responsive' to make it responsive, so its the same as the current screen